Actions/VerifyPRChanges/VerifyPRChanges.ps1

```diff
@@ -0,0 +1,34 @@
+Param(
+  [Parameter(HelpMessage = "Base commit of the PR", Mandatory = $false)]
+  [string] $baseSHA,
+  [Parameter(HelpMessage = "Head commit of the PR", Mandatory = $false)]
+  [string] $headSHA,
+  [Parameter(HelpMessage = "The name of the repository the PR is going to", Mandatory = $false)]
+  [string] $prBaseRepository,
+  [Parameter(HelpMessage = "The URL of the GitHub API", Mandatory = $false)]
+  [string] $githubApiUrl,
+  [Parameter(HelpMessage = "The token to use for the GitHub API", Mandatory = $false)]
+  [string] $githubToken
```

Comment:
In other actions, the token is just called token and placed first in the order.
I think we should use the same name here (unless it is a different token - and I don't think it is)